### PR TITLE
Embedded Systems: Mitschriften 26.04.2021

### PR DESCRIPTION
**Pull request by Steve2955**

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- EMB-GUENTHER/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: EMB-GUENTHER/README.md [hunk 1]
```diff
@@ -75,3 +81,108 @@ Elektromotor, der zusätzlich mit einem Sensor zur Positionsbestimmung ausgestat
...
+|A|B|Z|
+|-|-|-|
+|0|0|0|
+|0|1|0|
+|1|0|0|
+|1|1|1|
+
+- Code: ´if(a && b)´
+- Elektrisch: Zwei Schalter in Reihe
+- Anwendung: Schutzfunktion durch Erfüllung zweier Bedingungen
```
Comment (ZeroPointMax) [inline EMB-GUENTHER/README.md:99]: Wahnsinn, Weltneuheit :D

**Review by ZeroPointMax — APPROVED** [review]
nice 😎

File: EMB-GUENTHER/README.md (hunk 1 above)
Comment (Steve2955) [reply EMB-GUENTHER/README.md:99]: @ZeroPointMax Kannst dir vorstellen, wie lange wir das mit dem Günther hergeleitet haben 😂

Comment (nukerxy) [reply EMB-GUENTHER/README.md:99]: Was zur Hölle macht ihr ?!

**Review by nukerxy — APPROVED** [review]

**Pull request merged**